Index: src/my-game/MyGame.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/my-game/MyGame.ts b/src/my-game/MyGame.ts
--- a/src/my-game/MyGame.ts	(revision dc2995211fc237e2c19a905cbcaec6ad2b612205)
+++ b/src/my-game/MyGame.ts	(revision 42390bb2f67eb2508e213d3df85c4456c6b05fdb)
@@ -2,10 +2,8 @@
import {MyFeatures} from "@/my-game/MyFeatures";

export class MyGame extends IgtGame {
-    // @TODO Update SAVE_KEY to something unique
-    protected readonly SAVE_KEY: string = 'igt-vue';
-    // @TODO Update TICK_DURATION to an appropriate value
-    protected readonly TICK_DURATION: number = 0.01;
+    protected readonly SAVE_KEY: string = '123ishaTest/igt-docs';
+    protected readonly TICK_DURATION: number = 0.1;
     features: MyFeatures;

     constructor(features: MyFeatures) {
